stress-ng.c

```diff
@@ -1659,6 +1674,8 @@ static void MLOCKED_TEXT stress_run(
 				switch (pid) {
 				case -1:
 					if (errno == EAGAIN) {
+						pr_err("Cannot fork: errno=%d (%s)\n",
+							errno, strerror(errno));
 						(void)shim_usleep(100000);
```

Comment:
I'd rather not have this message added as on low resource systems this message can spam the output when it spins trying to get a free process slot.